### DISCUSSION TITLE
Read ExtendedData from Text Object

### DISCUSSION THREAD

**Discussion by thaiguer** [DomCR/ACadSharp#491] [Q&A]
Hello,
I'm trying to read ExtendedData from some entities with the ObjectType.TEXT. But, the return is null.
Reading ExtendedData from ObjectType.INSERT works fine.

I know there is ExtendedData in the both objects (Text and Insert) because I verified with XDList command (lisp).

Is this not implemented yet? Or am I missing something?

**Comment by DomCR** [comment]
Hi @thaiguer,

The extended data is set for all `CadObject` types in the same way, recently I fixed an issue regarding the dxf format but if you are seeing extended data in the insert entity it should not be a problem.

Could you provide a file as an example?

**Comment by thaiguer** [comment]
Hello.
I had to rewrite some of my code, and I could make it work.
Once again, the library is well written and is working properly.

Thanks.
